### DISCUSSION TITLE
Material Web Additions - Additional Material Design Components for Web

### DISCUSSION THREAD

**Discussion by maicol07** [material-components/material-web#4199] [Show and tell]
Hi,
I've made this (little for now) library to provide some missing components in the current Material We. I'll try my best to be the most possible M3 compliant.
The components are written with the lit framework, just like Material Web, using the same folder/file structure and the same tag schema (`<md-component-name>`).

Components available (as of writing):
- Card
- Data table
- Layout grid

_It will likely be expanded in the future. Components that will be added officially in Material Web with feature parity will be removed._

Here are the links: [Docs](https://main--625eadb22bf40d003a32215a.chromatic.com/) - [Github](https://github.com/maicol07/material-web-additions).

I hope you like it 😁 
PRs and feedbacks are welcome!

**Comment by Aworldc** [comment]
WooHoo 🎉🎉🎉🎉

**Comment by asyncliz** [comment]
Awesome work! Love to see it :)
